app/server/api-generator/src/test/java/io/syndesis/server/api/generator/openapi/util/OpenApiModelParserTest.java

```diff
@@ -85,16 +85,40 @@ public void shouldNotReportIssuesWithSupportedVersions() {
     }
 
     @Test
-    public void shouldReportIssuesWithUnsupportedVersions() {
+    public void shouldReportIssuesWithUnsupportedV2Versions() {
         final OpenApiModelInfo validated = OpenApiModelParser.parse(
-            "{\"openapi\": \"3.0.0\", \"info\":{ \"title\": \"test\", \"version\": \"1\"}, \"paths\": { \"/api\": { \"get\": {\"responses\": { \"200\": { \"description\": \"OK\" }}}}}}",
+            "{\"swagger\": \"3.0\", \"info\":{ \"title\": \"test\", \"version\": \"1\"}, \"paths\": { \"/api\": { \"get\": {\"responses\": { \"200\": { \"description\": \"OK\" }}}}}}",
```

Comment:
Perhaps we test with `openapi` 4.0 then?

Reply:
I will be using two tests one for `swagger:1.0` and one for `openapi:4.0`